modules/core/remove_orphaned.py

```diff
@@ -27,24 +34,25 @@ def rem_orphaned(self):
         root_files = []
         orphaned_files = []
         excluded_orphan_files = []
-        orphaned_parent_path = set()
 
         if self.remote_dir != self.root_dir:
+            local_orphaned_dir = self.orphaned_dir.replace(self.remote_dir, self.root_dir)
             root_files = [
                 os.path.join(path.replace(self.remote_dir, self.root_dir), name)
                 for path, subdirs, files in os.walk(self.remote_dir)
                 for name in files
-                if self.orphaned_dir.replace(self.remote_dir, self.root_dir) not in path
+                if local_orphaned_dir not in path
             ]
         else:
             root_files = [
                 os.path.join(path, name)
                 for path, subdirs, files in os.walk(self.root_dir)
                 for name in files
-                if self.orphaned_dir.replace(self.root_dir, self.remote_dir) not in path
+                if self.orphaned_dir not in path
             ]
 
         # Get an updated list of torrents
+        logger.print_line("Locating orphan files", self.config.loglevel)
         torrent_list = self.qbt.get_torrents({"sort": "added_on"})
         for torrent in torrent_list:
```

Comment:
```suggestion
        torrent_files_and_save_path = []
        for torrent in torrent_list:
            torrent_files = []
            for torrent_files_dict in torrent.files:
                torrent_files.append(torrent_files_dict.name)
            torrent_files_and_save_path.append((torrent_files, torrent.save_path))
        torrent_files.extend(
            [
                fullpath for fullpathlist in self.pool.starmap(get_full_path_of_torrent_files, torrent_files_and_save_path)
                for fullpath in fullpathlist if fullpath not in torrent_files
            ]
        )
```

Reply:
I almost wonder if it would be more efficient to do each torrent in its own thread instead of each file but thats probably not the case if you have a bunch of torrents with hundreds of files so this is probably fine.